docs/process/release.md

```diff
@@ -53,13 +53,17 @@ In general, cherry picks for LTS should only be done if it meets one of the crit
 
 # Release
 
-## Before releasing
-
-Update `Angular` version in
-[`packages/schematics/angular/utility/latest-versions.ts`](https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/utility/latest-versions.ts)
-**and** [`package.json`](https://github.com/angular/angular-cli/blob/master/package.json).
-
-## Shepparding
+For each version to be released, check out the associated branch and:
+
+1. Increment the CLI version number in
+   [`package.json`](https://github.com/angular/angular-cli/blob/master/package.json).
+1. Update `Angular` version in
+   [`packages/schematics/angular/utility/latest-versions.ts`](https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/utility/latest-versions.ts)
+   to match the lastest framework patch version for the associated minor.
+   - This number _usually_ aligns with the CLI version **but not always**. Framework may have
+     required a hotfix release, which could desync the two version numbers. Check the latest
+     [`@angular/core` package](https://www.npmjs.com/package/@angular/core) or the most recent FW
```

Comment:
Sometimes there is a delay on the NPM website in showing the latest version.
I find `yarn info @angular/core dist-tags` most reliable.

Reply:
Done.